src/aeplugin_dev_mode_handler.erl

```diff
@@ -161,15 +164,23 @@ spend_form() ->
       {input, #{type => submit, value => <<"Spend">>}, []}
      ]}.
 
+rollback_form() ->
+    {form, #{action => <<"/rollback">>, method => get},
+     [{label, #{for => height}, <<"To height: ">>},
+      {input, #{type => text, id => height, name => height}, []},
+      {label, #{for => hash}, <<"To hash: ">>},
+      {input, #{type => text, id => hash, name => hash}, []},
+      {input, #{type => submit, value => <<"Rollback">>}, []}
+     ]}.
+
 maybe_strong(true , Text) -> {strong, Text};
 maybe_strong(false, Text) -> Text.
 
 serve_request(#{path := <<"/emit_kb">>, qs := Qs}) ->
     Params = httpd:parse_query(Qs),
```

Comment:
It would be better to use the helper functions in `cowboy_req` instead, but perhaps FFR.